redbot/cogs/mod/kickban.py

```diff
@@ -246,7 +277,7 @@ async def kick(self, ctx: commands.Context, user: discord.Member, *, reason: str
     async def ban(
         self,
         ctx: commands.Context,
-        user: discord.Member,
+        user: Union[discord.Member, discord.User, RawUserIds],
```

Comment:
See the comment below for more information.
```suggestion
        user: Union[discord.Member, RawUserIds],
```

Reply:
As explained in the server, `discord.User` converter here as well as handling for it in `ban_user()` should be removed.